config/initializers/lograge.rb

```diff
@@ -2,4 +2,6 @@
 
 Rails.application.configure do
   config.lograge.enabled = Rails.env.production?
+  config.lograge.base_controller_class = ['ActionController::API', 'ActionController::Base']
```

Comment:
Patch both sdr-api's controllers and the DirectUploadsController to log less verbosely.